src/main/java/org/spongepowered/asm/mixin/gen/AccessorInfo.java

```diff
@@ -402,6 +397,13 @@ public static String inflectTarget(AccessorName name, AccessorType type, String
         return null;
     }
 
+    /**
+     * Get the type for this accessor
+     */
+    public AccessorType getType() {
```

Comment:
This is unnecessary.

Reply:
Getting the type is required to differentiate between getter and setter accessors.

Reply:
See below.

Reply:
I was under the impression this implementation would be desirable, as the alternative is to re-implement `initType` by checking `getReturnType().equals(Type.VOID_TYPE)` which seems messier.